tests/integration_test/access_log_test.go

```diff
@@ -61,39 +62,64 @@ func TestAccessLog(t *testing.T) {
 	// https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage#command-operators
 	accessLogFormat := "\"%REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)% %PROTOCOL%\"" +
 		"%RESPONSE_CODE% %RESPONSE_FLAGS% %BYTES_RECEIVED% %BYTES_SENT%" +
-		"\"%REQ(X-FORWARDED-FOR)%\" \"%REQ(USER-AGENT)%\"" +
-		"\"%REQ(:AUTHORITY)%\" \"%UPSTREAM_HOST%\" " +
-		"%FILTER_STATE(com.google.espv2.filters.http.path_matcher.operation):60% " +
+		"\"%REQ(X-FORWARDED-FOR)%\" \"%REQ(USER-AGENT)%\" " +
+		"%FILTER_STATE(com.google.espv2.filters.http.path_matcher.operation):70% " +
 		"%FILTER_STATE(com.google.espv2.filters.http.service_control.api_key):30% " +
-		"%FILTER_STATE(com.google.espv2.filters.http.service_control.path):60%" +
+		"%FILTER_STATE(com.google.espv2.filters.http.path_matcher.path):60%" +
```

Comment:
Such filter_state name is too much implemenation detail.   If we remove path_matcher,   it will break customers.

Reply:
ACK.